legos/cryptocurrency.py

```diff
@@ -24,12 +37,125 @@ def handle(self, message):
             logger.error('''Could not identify message source in message:
                         {}'''.format(message))
 
+        message_list = message['text'].split()
+        if message_list[0] == '!hodl':
+            self.reply(message, self._parse_multi_commands(message_list), opts)
+        else:
+            try:
+                query = message['text'].split()[1]
+            except:
```

Comment:
Since you're splitting and looking for an index here, the exception you're almost certainly going to raise is an `IndexError`, which is what I would catch. Upon catching the index error, I'd then return a helpful message indicating as much.


```
try:
    query: message['text'].split()[1]
except IndexError:
    self.reply(message, "Not enough arguments or something", opts)
```

Reply:
Resolved in 9aecef0bafb550b7bd2e50b75583a96fcfb1a457
and
090a524ea4d7c10997ae7625b992cd5410ec8fd2